sevntu-checks/ant-phase-verify-sevntu.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <project name="phase-package" default="execute">
-
-  <condition property="checkstyle.ant.skip">
-    <isset property="checkstyle.ant.skip" />
+  <condition property="run">
+    <or>
+        <not>
+            <isset property="checkstyle.ant.skip"/>
+        </not>
+        <isfalse value="${checkstyle.ant.skip}"/>
```

Comment:
Property run is set to true, if:
1) property `checkstyle.ant.skip` is not set
2) or property `checkstyle.ant.skip` is false